code/parachain/frame/staking-rewards/src/lib.rs

```diff
@@ -1281,71 +1306,62 @@ pub mod pallet {
 			reward_currency: Self::AssetId,
 			reward_increment: Self::Balance,
 		) -> DispatchResult {
-			RewardPools::<T>::try_mutate(pool, |reward_pool| {
-				match reward_pool {
-					Some(reward_pool) => {
-						match reward_pool.rewards.get_mut(&reward_currency) {
-							Some(mut reward) => {
-								let new_total_reward =
-									reward.total_rewards.safe_add(&reward_increment)?;
-								ensure!(
-									(new_total_reward
-										.safe_sub(&reward.total_dilution_adjustment)?) <=
-										reward.max_rewards,
-									Error::<T>::MaxRewardLimitReached
-								);
-								reward.total_rewards = new_total_reward;
-								let pool_account = Self::pool_account_id(pool);
-								T::Assets::transfer(
-									reward_currency,
-									from,
-									&pool_account,
-									reward_increment,
-									false,
-								)?;
-							},
-							None => {
-								// new reward asset so only pool owner is allowed to add.
-								// TODO (vim): consider enabling this later
-								// ensure!(
-								// 	*from == reward_pool.owner,
-								// 	Error::<T>::OnlyPoolOwnerCanAddNewReward
-								// );
-								let reward = Reward {
-									total_rewards: reward_increment,
-									claimed_rewards: Zero::zero(),
-									total_dilution_adjustment: T::Balance::zero(),
-									max_rewards: max(reward_increment, DEFAULT_MAX_REWARDS.into()),
-									reward_rate: RewardRate {
-										amount: T::Balance::zero(),
-										period: RewardRatePeriod::PerSecond,
-									},
-									last_updated_timestamp: 0,
-								};
-								reward_pool
-									.rewards
-									.try_insert(reward_currency, reward)
-									.map_err(|_| Error::<T>::TooManyRewardAssetTypes)?;
-								let pool_account = Self::pool_account_id(pool);
-								T::Assets::transfer(
-									reward_currency,
-									from,
-									&pool_account,
-									reward_increment,
-									false,
-								)?;
-							},
-						}
-						Self::deposit_event(Event::RewardTransferred {
-							from: from.clone(),
-							pool_id: *pool,
-							reward_currency,
-							reward_increment,
-						});
-						Ok(())
-					},
-					None => Err(Error::<T>::RewardsPoolNotFound.into()),
-				}
+			RewardPools::<T>::try_mutate(pool, |reward_pool| match reward_pool {
```

Comment:
Is there any change here other than the indentation?

Reply:
Just the removal of your TODO as discussed on slack.
Not sure why everything else got changed

Reply:
As long as you haven't made any change in the logic we are good.